trade-finance-manager-ui/server/controllers/case/cancellation/bank-request-date.controller.get.test.ts

```diff
@@ -0,0 +1,140 @@
+import { createMocks } from 'node-mocks-http';
+import { DEAL_SUBMISSION_TYPE } from '@ukef/dtfs2-common';
+import { aTfmSessionUser } from '../../../../test-helpers';
+import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
+import { BankRequestDateViewModel } from '../../../types/view-models';
+import { getBankRequestDate, GetBankRequestDateRequest } from './bank-request-date.controller';
+import api from '../../../api';
+
+jest.mock('../../../api', () => ({
+  getDeal: jest.fn(),
+  getDealCancellation: jest.fn(),
+}));
+
+const dealId = 'dealId';
+const ukefDealId = 'ukefDealId';
+const mockUser = aTfmSessionUser();
+
+describe('getBankRequestDate', () => {
```

Comment:
I've split the tests out into a `.get` and `.post` test file, which is why it looks like lots of new tests